client/src/components/_common/Form/FileInputDropZone.scss

```diff
@@ -10,7 +10,7 @@
   justify-content: center;
   align-items: center;
 
-  .icon-upload {
+  .icon.icon-upload {
```

Comment:
Which rule(s) needs this extra specificity?

_Local dev environment down[^1], but I tried the markup change on https://dev.cep.tacc.utexas.edu/ "New Ticket" modal, and did not need the extra `.icon` added to CSS._

<details><summary>If there are rules that need the extra specificity, then…</summary>

I'd consider writing the code so it is clearer which rules need the specificity, e.g. (a) separating those rules out and only giving them extra specificity, (b) adding a comment on the problem rule(s).

</details>

[^1]: I promise I don't make this up when I start a PR review late in the day. It really happens often.